lib/Support/Handlebars.cpp

```diff
@@ -131,7 +131,7 @@ escapeExpression(
     std::string_view str,
     HandlebarsOptions opt)
 {
-    if (opt.noHTMLEscape)
+    if (opt.noEscape)
```

Comment:
Please auto-close the issue as "fix" in the commit message

Reply:
OK. Is that all?

Reply:
for the tests yes but I need to look at the source tree refactoring